data/virt_autotest/host_autoyast/autoyast_opensuse_kvm_sshd.xml

```diff
@@ -0,0 +1,143 @@
+  </partitioning>
+  <timezone>
+    <hwclock>UTC</hwclock>
+    <timezone>America/New_York</timezone>
+  </timezone>
+  <networking>
+    <keep_install_network t="boolean">true</keep_install_network>
+    <backend>network_manager</backend>
+  </networking>
+  <software>
+    <products t="list">
+      <product>openSUSE</product>
+    </products>
+    <install_recommended t="boolean">true</install_recommended>
+    <patterns t="list">
+      <pattern>base</pattern>
+      <pattern>basesystem</pattern>
+      <pattern>enhanced_base</pattern>
+      <pattern>kvm_server</pattern>
+      <pattern>kvm_tools</pattern>
+    </patterns>
+    <packages t="list">
+      <package>autoyast2</package>
+      <package>supportutils</package>
+      <package>nmap</package>
+    </packages>
+  </software>
+  <users t="list">
+    <user>
+      <username>bernhard</username>
+      <user_password>$y$j9T$xsY9ubdI0jRF5JoSV8vvg1$03I5jIewyiCMm/EmkgRtYxvEYm83ZSJo0tl1rKP8Ad.</user_password>
+      <fullname>Bernhard M. Wiedemann</fullname>  
+      <encrypted t="boolean">true</encrypted>
+    </user>
+    <user>
+      <username>root</username>
+      <user_password>$y$j9T$Bc3oHTMRcmYxyzTlxEd7A/$jU0KChUccTna2tH9ItMRyRHC9fofibG37RGH4XkCxYD</user_password>
+      <encrypted t="boolean">true</encrypted>
+    </user>
+  </users>
+  <services-manager>
+    <default_target>multi-user</default_target>
+    <services>
+      <disable t="list">
+        <service>firewalld</service>
+      </disable>
+      <enable t="list">
+        <service>sshd</service>
+	<service>virtqemud</service>
+	<service>virtstoraged</service>
+	<service>virtnetworkd</service>
+	<service>virtnodedevd</service>
+      </enable>
+    </services>
+  </services-manager>
+  <scripts>
+    <!-- permit root login, password login and pubkeys login -->
+    <!-- keep ssh session alive -->
+    <!-- /etc/ssh/sshd_config is not default installed in TW -->
+    <init-scripts t="list">
+      <script>
+        <source><![CDATA[
+sshd_config_file="/etc/ssh/sshd_config"
+if [ ! -f $sshd_config_file ]; then
+    mkdir -p `dirname $sshd_config_file`
+    echo -e "PermitRootLogin yes\nPubkeyAuthentication yes\nPasswordAuthentication yes" > $sshd_config_file
+else
+    echo -e "TCPKeepAlive yes\nClientAliveInterval 60\nClientAliveCountMax 480" > $sshd_config_file
+    keys="PermitRootLogin PubkeyAuthentication PasswordAuthentication TCPKeepAlive"
+    for key in $keys; do
+        sed -i "/^[# ]*$key */{h;s/^[# ]*$key *.*\$/$key yes/};\${x;/^\$/{s//$key yes/;H};x}" $sshd_config_file
+	sed -i "/^[# ]*ClientAliveInterval */{h;s/^[# ]*ClientAliveInterval *.*\$/ClientAliveInterval 60/};\${x;/^\$/{s//ClientAliveInterval 60/;H};x}" $sshd_config_file
+        sed -i "/^[# ]*ClientAliveCountMax */{h;s/^[# ]*ClientAliveCountMax *.*\$/ClientAliveCountMax 480/};\${x;/^\$/{s//ClientAliveCountMax 480/;H};x}" $sshd_config_file 
+    done
+		fi
+]]>
+        </source>
+      </script>
+    </init-scripts>
+  </scripts>
+</profile>
```

Comment:
Can we just change the folder name to ```host_unattended_installation_files``` ?

Reply:
No problem, to be inline with guest autoyast profiles.